TaskHero/TaskCatalogDetailViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  TaskCatalogDetailViewController.swift
+//  TaskHero
+//
+//  Created by Akifumi Shinagawa on 11/26/16.
+//  Copyright © 2016 Task Hero. All rights reserved.
+//
+
+import UIKit
+
+class TaskCatalogDetailViewController: UIViewController {
+
+    @IBOutlet weak var taskName: UILabel!
+    @IBOutlet weak var taskDetail: UILabel!
+    @IBOutlet weak var taskEstimatedTime: UILabel!
+    @IBOutlet weak var tableView: UITableView!
+    
+    fileprivate let stepCellIdentifier = "EditStepCell"
+    var currentSelectedCellRowNum = -1
+    
+    var steps:[Step]!
+    var task:Task? {
+        didSet{
+            steps = task?.steps
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        taskName.text = task?.name
+        taskDetail.text = task?.details
+        taskEstimatedTime.text = String(describing: (task?.estimatedTime)!)
+        
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.rowHeight = UITableViewAutomaticDimension
+        tableView.estimatedRowHeight = 160
+        tableView.register(UINib(nibName: stepCellIdentifier, bundle: nil), forCellReuseIdentifier: stepCellIdentifier)
```

Comment:
You can use this extension function to register the cells https://github.com/Task-Hero/TaskHero-iOS/blob/master/TaskHero/UITableView%2BConvenience.swift